app/src/main/java/com/example/android/psiak/Firebase/BasePresenter.java

```diff
@@ -0,0 +1,21 @@
+package com.example.android.psiak.Firebase;
+
+/**
+ * Created by Grzegorz on 23.11.2017.
```

Comment:
Move it outside Firebase package since it is the base class for all presenters in the application.